typings/components/ChipsInput.d.ts

```diff
@@ -1,7 +1,9 @@
+import { TypographyModifiers } from 'generatedTypes';
 import {ReactElement} from 'react';
 import {StyleProp, TextStyle, ViewStyle, AccessibilityProps, ColorValue} from 'react-native';
 import {BaseComponent} from '../commons';
 import {AccessibilityProps} from '../modifiers';
+import { TextInputProps } from './Inputs';
```

Comment:
Shouldn't this be `TextFieldProps`?

Reply:
Hmm, I don't really know as I've only seen this used in our module as `TextInput` you know the use cases better than me :)

Reply:
Fixed

Reply:
> you know the use cases better than me :)
Not really, it's just using a `TextField` with `{...others}` :)